### PR TITLE
[github] Update code owners file

### PR DESCRIPTION
**Pull request by tsapeta**
# Why

We'd like to have at least two owners of each unimodule, so it will be easier to cooperate in following aspects:
- Bug fixing
- Discussing ongoing changes
- Creating new functionalities
- Code reviewing

# How

- Added some missing packages to the list.
- Added second owner to packages where there is no doubt who should be the owner.
- Put a hyphen to mark orphaned packages.

**Please use GitHub's pull request suggestions if you want to take care or opt out from a package!** 🙏 

# Test Plan

Not necessary.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,35 +1,38 @@
-/packages/@unimodules                           @sjchmiela
+/packages/@unimodules/core                      @sjchmiela @tsapeta
+/packages/@unimodules/react-native-adapter      @sjchmiela @tsapeta
 /packages/babel-preset-expo                     @brentvatne
-/packages/expo                                  @ide @tsapeta
 /packages/expo-ads-admob                        @sjchmiela
-/packages/expo-ads-facebook                     @Szymon20000
+/packages/expo-ads-facebook                     -
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-ads-facebook                     @sjchmiela
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,35 +1,38 @@
-/packages/@unimodules                           @sjchmiela
+/packages/@unimodules/core                      @sjchmiela @tsapeta
+/packages/@unimodules/react-native-adapter      @sjchmiela @tsapeta
```
Comment (sjchmiela) [inline .github/CODEOWNERS]: Wouldn't this work too?
```suggestion
/packages/@unimodules                           @sjchmiela @tsapeta
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -41,37 +44,42 @@
...
 /packages/expo-print                            @tsapeta
 /packages/expo-processing                       @EvanBacon
 /packages/expo-random                           @EvanBacon
 /packages/expo-secure-store                     @sjchmiela
 /packages/expo-sensors                          @sjchmiela
 /packages/expo-sharing                          @sjchmiela
 /packages/expo-sms                              @bbarthec
-/packages/expo-speech                           @Szymon20000
-/packages/expo-sqlite                           @Szymon20000
+/packages/expo-speech                           -
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-speech                           @sjchmiela
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -41,37 +44,42 @@
...
+/packages/expo-sqlite                           -
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-sqlite                           @sjchmiela 
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -41,37 +44,42 @@
...
+/packages/expo-location                         @tsapeta @lukmccall
 /packages/expo-mail-composer                    @sjchmiela
-/packages/expo-media-library                    @tsapeta
+/packages/expo-media-library                    @tsapeta @lukmccall
 /packages/expo-module-scripts                   @ide
 /packages/expo-module-template                  @bbarthec
 /packages/expo-network                          @ide
-/packages/expo-payments-stripe                  @Szymon20000
-/packages/expo-permissions                      @bbarthec
+/packages/expo-payments-stripe                  -
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-payments-stripe                  @sjchmiela
```

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -1,35 +1,38 @@
...
-/packages/expo-ads-facebook                     @Szymon20000
+/packages/expo-ads-facebook                     -
 /packages/expo-analytics-amplitude              @sjchmiela
 /packages/expo-analytics-segment                @sjchmiela
 /packages/expo-app-auth                         @EvanBacon
 /packages/expo-app-loader-provider              @tsapeta
+/packages/expo-apple-authentication             @esamelson @tsapeta
 /packages/expo-application                      @ide
 /packages/expo-asset                            @tsapeta
 /packages/expo-av                               @sjchmiela @mczernek
```
Comment (sjchmiela) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-av                               @mczernek
```
😈

File: .github/CODEOWNERS (hunk 2 above)
Comment (tsapeta) [reply .github/CODEOWNERS]: But why? :P I'd prefer having each package listed here separately

Comment (sjchmiela) [reply .github/CODEOWNERS]: If we ever add a new package to `@unimodules` we wouldn't have to customize this file. I'm also ok with leaving this as it is.

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -1,35 +1,38 @@
...
+/packages/expo-ads-facebook                     @sjchmiela
 /packages/expo-analytics-amplitude              @sjchmiela
 /packages/expo-analytics-segment                @sjchmiela
 /packages/expo-app-auth                         @EvanBacon
 /packages/expo-app-loader-provider              @tsapeta
+/packages/expo-apple-authentication             @esamelson @tsapeta
 /packages/expo-application                      @ide
 /packages/expo-asset                            @tsapeta
-/packages/expo-av                               @sjchmiela @mczernek
+/packages/expo-av                               @mczernek
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-av                               @mczernek @bbarthec
```

File: .github/CODEOWNERS [hunk 8]
```diff
@@ -1,35 +1,38 @@
...
+/packages/expo-battery                          -
+/packages/expo-bluetooth                        @EvanBacon
 /packages/expo-blur                             @sjchmiela
 /packages/expo-branch                           @sjchmiela
 /packages/expo-brightness                       @sjchmiela
-/packages/expo-calendar                         @tsapeta
+/packages/expo-calendar                         @tsapeta @lukmccall
 /packages/expo-camera                           @bbarthec
 /packages/expo-cellular                         @ide
 /packages/expo-constants                        @tsapeta
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-constants                        @tsapeta @bbarthec 
```

File: .github/CODEOWNERS [hunk 9]
```diff
@@ -1,35 +1,38 @@
...
 /packages/expo-contacts                         @EvanBacon
 /packages/expo-crypto                           @EvanBacon
 /packages/expo-device                           @ide
 /packages/expo-document-picker                  @bbarthec
+/packages/expo-error-recovery                   @lukmccall
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-error-recovery                   @lukmccall @bbarthec 
```

File: .github/CODEOWNERS [hunk 10]
```diff
@@ -1,35 +1,38 @@
...
 /packages/expo-face-detector                    @sjchmiela @mczernek
 /packages/expo-facebook                         @sjchmiela
 /packages/expo-file-system                      @sjchmiela
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-file-system                      @sjchmiela @bbarthec 
```

File: .github/CODEOWNERS [hunk 11]
```diff
@@ -1,35 +1,38 @@
...
 /packages/expo-font                             @sjchmiela
-/packages/expo-font-interface                   @sjchmiela
-/packages/expo-gl                               @tsapeta
 /packages/expo-gl-cpp                           @tsapeta
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-gl-cpp                           @tsapeta @bbarthec 
```

File: .github/CODEOWNERS [hunk 12]
```diff
@@ -1,35 +1,38 @@
...
+/packages/expo-gl                               @tsapeta
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-gl                               @tsapeta @bbarthec 
```

File: .github/CODEOWNERS [hunk 13]
```diff
@@ -1,35 +1,38 @@
...
 /packages/expo-google-app-auth                  @EvanBacon
 /packages/expo-google-sign-in                   @EvanBacon
 /packages/expo-haptics                          @tsapeta
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-haptics                          @tsapeta @bbarthec 
```

File: .github/CODEOWNERS [hunk 14]
```diff
@@ -41,37 +44,42 @@
 /packages/expo-linear-gradient                  @brentvatne
 /packages/expo-local-authentication             @tsapeta
 /packages/expo-localization                     @EvanBacon
-/packages/expo-location                         @tsapeta
+/packages/expo-location                         @tsapeta @lukmccall
 /packages/expo-mail-composer                    @sjchmiela
-/packages/expo-media-library                    @tsapeta
+/packages/expo-media-library                    @tsapeta @lukmccall
 /packages/expo-module-scripts                   @ide
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-module-scripts                   @ide @bbarthec 
```

File: .github/CODEOWNERS [hunk 15]
```diff
@@ -41,37 +44,42 @@
...
-/packages/expo-payments-stripe                  @Szymon20000
-/packages/expo-permissions                      @bbarthec
+/packages/expo-payments-stripe                  @sjchmiela
+/packages/expo-permissions                      @bbarthec @lukmccall
 /packages/expo-print                            @tsapeta
 /packages/expo-processing                       @EvanBacon
 /packages/expo-random                           @EvanBacon
 /packages/expo-secure-store                     @sjchmiela
 /packages/expo-sensors                          @sjchmiela
 /packages/expo-sharing                          @sjchmiela
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-sharing                          @sjchmiela @bbarthec 
```

File: .github/CODEOWNERS [hunk 16]
```diff
@@ -41,37 +44,42 @@
...
-/packages/expo-updates                          @mczernek
+/packages/expo-updates                          @esamelson @mczernek
 /packages/expo-video-thumbnails                 @sjchmiela
 /packages/expo-web-browser                      @sjchmiela @mczernek
 /packages/expo-yarn-workspaces                  @ide
+/packages/expo                                  @ide @tsapeta
+/packages/jest-expo-enzyme                      @ide @EvanBacon
+/packages/jest-expo-puppeteer                   @ide @EvanBacon
 /packages/jest-expo                             @brentvatne
+/packages/react-native-unimodules               @tsapeta
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/react-native-unimodules               @tsapeta @bbarthec 
```

File: .github/CODEOWNERS [hunk 17]
```diff
@@ -41,37 +44,42 @@
...
 /packages/unimodules-barcode-scanner-interface  @bbarthec
 /packages/unimodules-camera-interface           @bbarthec
 /packages/unimodules-constants-interface        @tsapeta
 /packages/unimodules-face-detector-interface    @sjchmiela
 /packages/unimodules-file-system-interface      @sjchmiela
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/unimodules-file-system-interface      @sjchmiela @bbarthec 
```

File: .github/CODEOWNERS [hunk 18]
```diff
@@ -41,37 +44,42 @@
...
 /packages/expo-web-browser                      @sjchmiela @mczernek
 /packages/expo-yarn-workspaces                  @ide
+/packages/expo                                  @ide @tsapeta
+/packages/jest-expo-enzyme                      @ide @EvanBacon
+/packages/jest-expo-puppeteer                   @ide @EvanBacon
 /packages/jest-expo                             @brentvatne
+/packages/react-native-unimodules               @tsapeta
 /packages/unimodules-barcode-scanner-interface  @bbarthec
 /packages/unimodules-camera-interface           @bbarthec
 /packages/unimodules-constants-interface        @tsapeta
```
Comment (bbarthec) [inline .github/CODEOWNERS]:
```suggestion
/packages/unimodules-constants-interface        @tsapeta @bbarthec 
```

File: .github/CODEOWNERS [hunk 19]
```diff
@@ -41,37 +44,42 @@
...
+/packages/expo-module-scripts                   @ide @bbarthec 
 /packages/expo-module-template                  @bbarthec
 /packages/expo-network                          @ide
-/packages/expo-payments-stripe                  @Szymon20000
-/packages/expo-permissions                      @bbarthec
+/packages/expo-payments-stripe                  @sjchmiela
+/packages/expo-permissions                      @bbarthec @lukmccall
 /packages/expo-print                            @tsapeta
 /packages/expo-processing                       @EvanBacon
 /packages/expo-random                           @EvanBacon
```
Comment (tsapeta) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-random                           @EvanBacon
/packages/expo-screen-orientation               @lukmccall
```

File: .github/CODEOWNERS [hunk 20]
```diff
@@ -1,38 +1,41 @@
...
-/packages/expo-gl                               @tsapeta
-/packages/expo-gl-cpp                           @tsapeta
+/packages/expo-gl-cpp                           @tsapeta @bbarthec 
+/packages/expo-gl                               @tsapeta @bbarthec 
 /packages/expo-google-app-auth                  @EvanBacon
 /packages/expo-google-sign-in                   @EvanBacon
-/packages/expo-haptics                          @tsapeta
+/packages/expo-haptics                          @tsapeta @bbarthec 
 /packages/expo-image-manipulator                @bbarthec
 /packages/expo-image-picker                     @bbarthec
```
Comment (lukmccall) [inline .github/CODEOWNERS]:
```suggestion
/packages/expo-image-picker                     @bbarthec @lukmccall
```

**Pull request merged**